src/widgets/conditions/condition-record.component.tsx

```diff
@@ -32,14 +32,9 @@ export default function ConditionRecord(props: ConditionRecordProps) {
           styles={{ width: "100%" }}
           editComponent={ConditionsForm}
           showComponent={() => {
-            openEditConditionsWorkspaceTab(
-              ConditionsForm,
-              "Edit Conditions",
-              patientCondition?.id,
-              patientCondition?.code?.text,
-              patientCondition?.clinicalStatus,
-              patientCondition?.onsetDateTime
-            );
+            openWorkspaceTab(ConditionsForm, "Edit Conditions", {
+              conditionUuid: patientCondition?.id
```

Comment:
Is that equivalent? Why are the props like the clinical status etc. not required any more?

(Just for my understanding - technically this seems to be alright)

Reply:
I'm making a request in `ConditionsForm` component to fetch a condition using the provided uuid. I could as well pass the data required from the props in `ConditionRecord`. Which approach would you prefer?

Reply:
ping @FlorianRappl

Reply:
Sorry @denniskigen - I lost my track here.

Back to your question: If the data is already available we should just pass it on (instead of making another request). Otherwise, having the request only in `ConditionsForm` makes sense to me. So if its the latter - leave it, if we do have all data then pass it on.

Reply:
No worries. We have all the data. I'll refactor it so that it passes on the data then. Thanks.

Reply:
Great thanks! :+1: